fabtests/common/shared.c

```diff
@@ -3556,3 +3563,31 @@ void ft_free_string_array(char **s)
 	/* and then the actual array of pointers */
 	free(s);
 }
+
+int ft_pin_core(int core_id)
```

Comment:
This function can be static

Reply:
Do we want to limit it to single core pinning? Tools like taskset allow pinning to a list/range of cores, like '0-7,16-23'. That would be especially useful if we want to choose a specific CPU socket instead of a specific core.

Reply:
Fixed, and also enabled the option to accept a core_list, its format uses a comma-separate list.